index.js

```diff
@@ -4,12 +4,29 @@
 module.exports = {
     name: 'ember-cli-hot-loader',
     serverMiddleware: function (config){
-        var lsReloader = require('./lib/hot-reloader')(config.options);
-        lsReloader.run();
+        if (config.options.environment === 'development') {
+            var lsReloader = require('./lib/hot-reloader')(config.options);
+            lsReloader.run();
+        }
     },
     included: function (app) {
         this._super.included(app);
-        // TODO: consider removing this since it adds an unnecessary runtime dependency to all apps
-        app.import(app.bowerDirectory + '/ember/ember-template-compiler.js');
+
+        // If not in dev, bail
+        if (app.env !== 'development') {
```

Comment:
minor/later: we should make it configurable with `app.env !== 'development'` as a default.